cfgov/unprocessed/css/print.less

```diff
@@ -28,25 +28,38 @@
     - cfgov-print
 */
 
+.respond-to-print({
```

Comment:
Have you considered adding in any more print-appropriate base styles on body? eg `background: none` & `color: black` (which would also apply to the link color on line 48).

Also curious about your thoughts on `.o-header`, `.o-feedback`, `.content_sidebar`, and `.related-questions` being `display: none`d on print

Reply:
These are good questions, but ones that I think we should save for later, like some of the others that were mentioned later in the discussion.